pom.xml

```
@@ -96,11 +96,12 @@
         <maven-jar-plugin.version>3.1.0</maven-jar-plugin.version>
         <maven.enforced.version>3.0.0-M2</maven.enforced.version>
 
-        <kafka.version>3.0.0</kafka.version>
+        <kafka.version>3.3.1</kafka.version>
```

3.3.2? I guess we want to stick with the latest?

```suggestion
        <kafka.version>3.3.2</kafka.version>
```